### PR TITLE
Schema registry

### PR DESCRIPTION
**Pull request by sancar**
+ Connect plugins documentation
+ Connect AllowList as troubleshoot

### CODE REVIEW

File: kafka/schema-registry/schemahowto.mdx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+### Producer
+
+If you need to configure your producers to use the schema registry add following
+properties to consumer properties in addition to the broker configurations. 
+Note that the seleted deserializer needs to be schema-registry aware.
+
+```java
+Properties props = new Properties();
+// ... other configurations like broker.url and broker authentication is skipped
```
Comment (mdogan) [inline kafka/schema-registry/schemahowto.mdx:24]: _are_ skipped.

File: kafka/schema-registry/schemahowto.mdx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+{
+    "name": "myConnector",
+    "properties": {
+        "connector.class": "io.debezium.connector.mysql.MySqlConnector",
+        // other configurations are skipped.
+
+        "key.converter": "io.confluent.connect.avro.AvroConverter",
+        "key.converter.basic.auth.credentials.source": "USER_INFO",
+        "key.converter.basic.auth.user.info": "UPSTASH_KAFKA_REST_USERNAME:UPSTASH_KAFKA_REST_PASSWORD",
+        "key.converter.schema.registry.url": "UPSTASH_KAFKA_REST_URL/schemaRegistry",
```
Comment (mdogan) [inline kafka/schema-registry/schemahowto.mdx:88]: `/schemaRegistry` -> `/schema-registry`

File: kafka/schema-registry/schemahowto.mdx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+        "value.converter": "io.confluent.connect.avro.AvroConverter",
+        "value.converter.basic.auth.credentials.source": "USER_INFO",
+        "value.converter.basic.auth.user.info": "UPSTASH_KAFKA_REST_USERNAME:UPSTASH_KAFKA_REST_PASSWORD",
+        "value.converter.schema.registry.url": "UPSTASH_KAFKA_REST_URL/schemaRegistry"
```
Comment (mdogan) [inline kafka/schema-registry/schemahowto.mdx:93]: `/schemaRegistry` -> `/schema-registry`

File: kafka/schema-registry/schemaintroduction.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: Introduction
+---
+
+TODO sancar trademark ?
```
Comment (mdogan) [inline kafka/schema-registry/schemaintroduction.mdx:5]: No trademarks exist afaik.

File: kafka/schema-registry/schemaintroduction.mdx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+Schema Registry serves as a central hub to handle and validate schemas for message data related to Kafka topics. 
+It also manages serialization and deserialization of data over the network. 
+This aids producers and consumers in maintaining data consistency and compatibility as schemas change. 
+
+TODO sancar, should we mention Confluent compatibility at all ? I feel like we can skip this part.
```
Comment (mdogan) [inline kafka/schema-registry/schemaintroduction.mdx:11]: Not sure. @enesakar ?

Comment (sancar) [reply kafka/schema-registry/schemaintroduction.mdx]: I will mention it as we decided.

**Review by mdogan — APPROVED** [review]

**Pull request merged**